stories/module-analytics-settings.stories.js

```diff
@@ -0,0 +1,210 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { removeAllFilters, addFilter } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import SettingsModule from '../assets/js/components/settings/settings-module';
+import { SettingsMain as AnalyticsSettings } from '../assets/js/modules/analytics/settings';
+import { fillFilterWithComponent } from '../assets/js/util';
+import * as fixtures from '../assets/js/modules/analytics/datastore/__fixtures__';
+
+import { STORE_NAME } from '../assets/js/modules/analytics/datastore';
+import { WithTestRegistry } from '../tests/js/utils';
+
+function filterAnalyticsSettings() {
+	// set( global, 'googlesitekit.modules.analytics.setupComplete', true );
+	removeAllFilters( 'googlesitekit.ModuleSettingsDetails-analytics' );
+	addFilter(
+		'googlesitekit.ModuleSettingsDetails-analytics',
+		'googlesitekit.AnalyticsModuleSettings',
+		fillFilterWithComponent( AnalyticsSettings )
+	);
+}
+
+function Settings( props ) {
+	const {
+		callback,
+		module = global.googlesitekit.modules.analytics,
+		isEditing = false,
+		isOpen = true,
+		isSaving = false,
+		error = false,
+		// eslint-disable-next-line no-console
+		handleAccordion = ( ...args ) => console.log( 'handleAccordion', ...args ),
+		// eslint-disable-next-line no-console
+		handleDialog = ( ...args ) => console.log( 'handleDialog', ...args ),
+		// eslint-disable-next-line no-console
+		updateModulesList = ( ...args ) => console.log( 'updateModulesList', ...args ),
+		// eslint-disable-next-line no-console
+		handleButtonAction = ( ...args ) => console.log( 'handleButtonAction', ...args ),
+	} = props;
+
+	return (
+		<WithTestRegistry callback={ callback }>
+			<div style={ { background: 'white' } }>
+				<SettingsModule
+					key={ module.slug + '-module' }
+					slug={ module.slug }
+					name={ module.name }
+					description={ module.description }
+					homepage={ module.homepage }
+					learnmore={ module.learnMore }
+					active={ module.active }
+					hasSettings={ true }
+					autoActivate={ module.autoActivate }
+					updateModulesList={ updateModulesList }
+					handleEdit={ handleButtonAction }
+					handleConfirm
+					isEditing={ isEditing ? { 'analytics-module': true } : {} }
+					isOpen={ isOpen }
+					handleAccordion={ handleAccordion }
+					handleDialog={ handleDialog }
+					provides={ module.provides }
+					isSaving={ isSaving }
+					screenID={ module.screenID }
+					error={ error }
+				/>
+			</div>
+		</WithTestRegistry>
+	);
+}
+
+storiesOf( 'Analytics Module Settings', module )
```

Comment:
Let's change this to `Analytics Module/Settings`. This will make `Settings` be a separate "sub-story" of `Analytics Module`, which makes sense here.

Reply:
I didn't realize stories could be organized like this. Awesome! 👍